### PR TITLE
fix compile error

### PR DESCRIPTION
**Pull request by 17Skye17**
keyword argument 'average_window' is not expected

### CODE REVIEW

**Comment by wangkuiyi** [comment]
I noticed that in https://github.com/PaddlePaddle/Paddle/pull/905/, @pengli09 removed the parameter `average_window`. Could @pengli09 please review this PR? Thanks!

File: demo/sentiment/trainer_config.py [hunk 1]
```diff
@@ -29,7 +29,6 @@
     batch_size=128,
     learning_rate=2e-3,
     learning_method=AdamOptimizer(),
-    average_window=0.5,
```
Comment (pengli09) [inline demo/sentiment/trainer_config.py]: This line should be changed into `model_average=ModelAverage(average_window=0.5),`

**Comment by wangkuiyi** [comment]
我看 @17Skye17 在某个PR里已经把 average_window=0.5 这行删掉了。这里还需要按照 @pengli09 的[建议](https://github.com/PaddlePaddle/Paddle/pull/1528#discussion_r104075353) 修改吗？还是这个PR可以关掉了？也请 @qingqing01 看看吧。

**Comment by qingqing01** [comment]
是需要按照 @pengli09 的建议修改，即参考 https://github.com/PaddlePaddle/Paddle/blob/develop/demo/semantic_role_labeling/db_lstm.py#L89  
另外，也推荐使用V2 API: https://github.com/PaddlePaddle/book/tree/develop/05.understand_sentiment

**Comment by qingqing01** [comment]
Close this PR, since it was fixed by #1882

**Pull request closed without merging**